Update: Feature 216 export articles as pdf

di72nn opened the pull request.

tcitworld commented on app/src/main/java/fr/gaulupeau/apps/Poche/network/WallabagConnection.java:
```diff
@@ -112,18 +112,18 @@ public static OkHttpClient getClient() {
     public static OkHttpClient createClient() {
-        OkHttpClient client = null;
+        OkHttpClient.Builder b = new OkHttpClient.Builder()
+                .readTimeout(45, TimeUnit.SECONDS);
```
I liked the idea of setting the timeout for connect, write and read, and even have them as parameters.

di72nn replied:
I removed `connect` and `write` timeouts because they were equal to the default values. I don't mind getting them back.

tcitworld replied:
Yeah this one is increased specifically to allow the download of big pdf files (which can take a lot of time to generate on the server's side). I actually don't know about the others, but I definitely want this as an advanced setting (well, when https://github.com/wallabag/android-app/issues/143 is done).

di72nn commented:
I'll add some more updates here soon.

di72nn commented:
Updated.
Added a simple notification that opens PDF file on touch (maybe the `exportDir` should be opened instead?).
I'm really not in the mood to handle all the messages so I left TODO's to extract strings. Maybe a couple of Toast messages should be added (on finish/fail).
